lib/UmdMainTemplatePlugin.js

```diff
@@ -164,7 +164,7 @@ UmdMainTemplatePlugin.prototype.apply = function(compilation) {
 				"		for(var i in a) (typeof exports === 'object' ? exports : root)[i] = a[i];\n" +
 				"	}\n"
 			) +
-			"})(this, function(" + externalsArguments(externals) + ") {\nreturn ", "webpack/universalModuleDefinition"), source, "\n});\n");
+			"})(this, function(" + externalsArguments(externals) + ") {\nreturn ", "webpack/universalModuleDefinition"), source, ";\n})");
```

Comment:
I thought the trailing `;` added by [`return new ConcatSource(source, ";");`] (https://github.com/webpack/webpack/blob/master/lib/MainTemplate.js#L168) by MainTemplate.js